mathesar_ui/src/header/schema-selector/SchemaSelector.scss

```diff
@@ -129,15 +129,23 @@ button.dropdown.trigger.selector {
     .schemas {
       flex-basis: 230px;
       border-left: 1px solid #efefef;
+      overflow: hidden;
 
       ul {
         height: 198px;
-        overflow: auto;
```

Comment:
Why is `overflow: auto` removed here?

When we have a large number of schemas, we would not be able to scroll if this property isn't present. Try adding more schemas and test it out.

It would be better to update this is `overflow-y: auto`.